src/main/java/org/mastodon/mamut/feature/branch/sinuosity/BranchSinuosityFeatureSerializer.java

```diff
@@ -0,0 +1,57 @@
+package org.mastodon.mamut.feature.branch.sinuosity;
+
+import static org.mastodon.mamut.feature.branch.sinuosity.BranchSinuosityFeature.BRANCH_SINUOSITY_FEATURE_SPEC;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import org.mastodon.feature.io.FeatureSerializer;
+import org.mastodon.io.FileIdToObjectMap;
+import org.mastodon.io.ObjectToFileIdMap;
+import org.mastodon.io.properties.DoublePropertyMapSerializer;
+import org.mastodon.mamut.feature.branch.BranchFeatureSerializer;
+import org.mastodon.mamut.model.ModelGraph;
+import org.mastodon.mamut.model.Spot;
+import org.mastodon.mamut.model.branch.BranchSpot;
+import org.mastodon.mamut.model.branch.ModelBranchGraph;
+import org.mastodon.properties.DoublePropertyMap;
+import org.scijava.plugin.Plugin;
+
+/**
+ * De-/serializes the {@link BranchSinuosityFeature}.
+ */
+@Plugin( type = FeatureSerializer.class )
+public class BranchSinuosityFeatureSerializer
```

Comment:
This class should extend `BranchSpotDoublePropertyFeatureSerializer`.

Reply:
Done.